karma.conf.js

```diff
@@ -13,8 +13,10 @@ module.exports = (config) => {
       'node_modules/pixi.js/dist/pixi.js',
       'node_modules/react-bootstrap/dist/react-bootstrap.js',
       'node_modules/bootstrap/dist/css/bootstrap.min.css',
+      'node_modules/font-awesome/css/font-awesome.css',
```

Comment:
If possible I would be great if we can avoid adding another big dependency :)

Reply:
Ok, it's gone.